programs/x509/cert_write.c

```diff
@@ -686,6 +746,8 @@ int main( int argc, char *argv[] )
         mbedtls_printf( " ok\n" );
     }
 
+    mbedtls_x509write_crt_set_md_alg( &crt, opt.sig_alg );
```

Comment:
line 708 has

```
mbedtls_x509write_crt_set_md_alg( &crt, opt.md );
```

so - given that `sig_alg` should really be `md_alg`, is all the `sig_alg` stuff necessary?

Reply:
Agreed, `sig_alg` looks like it simply duplicates the `md` argument. I've removed it.